src/main/typescript/components/Footer.tsx

```diff
@@ -25,13 +25,16 @@ const nwo = require("../../resources/img/footer/logo_NWO.png")
 
 const ContactColumn = () => (
     <ul className="contact">
-        <li>Anna van Saksenlaan 51</li>
-        <li>2593 HW Den Haag</li>
         <li>
+            <a href="tel:+31-703494450">+31 70 349 44 50</a>
+        </li>        <li>
             <a href="mailto:info@dans.knaw.nl">info@dans.knaw.nl</a>
         </li>
         <li>
-            <a title="View how to contact" href="http://dans.knaw.nl/en/contact" target="_blank">More ›››</a>
+            <a title="View how to contact" href="https://dans.knaw.nl/en/contact" target="_blank">More ›››</a>
```

Comment:
Switch privacy statement and `More >>>`. Seems more obvious to me.

Reply:
the more is about the contact info.
first, i put the privacy statement link in the third column, with the other legal links. 
however, on the dans website it is on the first column, and chris asked to put it there

Reply:
No, I meant to put the `More >>>` below the privacy statement.